spec/spec_helper.rb

```diff
@@ -129,13 +137,26 @@ def stub_create_branch_request(repo_name, branch_name)
     to_return(status: 200)
 end
 
+def stub_update_contents_request(repo_name, path:, content:, previous_content:, commit_title:, branch:)
+  stub_request(:put, "https://api.github.com/repos/alphagov/#{repo_name}/contents/#{path}").
+    with(
+      body: {
+        branch: branch,
+        content: Base64.strict_encode64(content),
+        message: commit_title,
+        sha: Digest::SHA1.hexdigest("blob #{previous_content.length}\0#{previous_content}")
+      }
+    ).
+    to_return(status: 200)
+end
+
 def stub_create_contents_request(repo_name, path:, content:, commit_title:, branch:)
   stub_request(:put, "https://api.github.com/repos/alphagov/#{repo_name}/contents/#{path}").
     with(
       body: {
-        "branch": branch,
-        "content": Base64.encode64(content).gsub(/\n/, ""),
-        "message": commit_title,
+        branch: branch,
+        content: Base64.strict_encode64(content),
```

Comment:
Why the switch to `strict_encode64`? Could you explain in the commit?

Reply:
It's because `encode64` adds line breaks after IIRC 80 columns, which I don't want.

I think it makes more sense to amend the original commit that introduced this code, so I'll do that